docs/notes/2.23.x.md

```diff
@@ -137,6 +137,9 @@ Fixed pulling `helm_artifact`s from OCI repositories.
 
 Added `workspace_invalidation_sources` field to `adhoc_tool` and `shell_command` target types. This new field allows declaring that these targets depend on files without bringing those files into the execution sandbox, but that the target should still be re-executed if those files change. This is intended to work with the `workspace_environment` support where processes are executed in the workspace and not in a separate sandbox.
 
+#### Semgrep
+Semgrep now allows configuring the config directory, ignore config file, and will recursively discover all rules in the directory.
```

Comment:
```suggestion
Semgrep now allows configuring the config directory, the ignore config file, and will recursively discover all rules in the directory.
```

I initially misread the meaning of "ignore config file", this makes it more clear that it is a thing, not that it is some config file being ignored.